src/main/java/org/vietabroader/view/verifier/ColumnVerifier.java

```diff
@@ -14,9 +14,12 @@ public boolean verify(JComponent input) {
     public boolean shouldYieldFocus(JComponent input) {
         boolean inputOK = verify(input);
         if (inputOK) {
+            input.setForeground(Color.black);
             return true;
         } else {
             Toolkit.getDefaultToolkit().beep();
+            input.setForeground(Color.red);
+            input.setToolTipText("Enter a valid column name. Ex: A, AB, ...");
```

Comment:
Sorry, I forgot that this kind of tooltip only appears when we hover on it. Let's move this setToolTipText to right after the button is created. Setting foreground color to red is a good idea.